.github/dependabot.yml

```
@@ -0,0 +1,10 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: monthly
```

```suggestion
    schedule:
      interval: monthly
    assignees:
      - "ezio-melotti"
```

I don't mind being pinged about these, as I already handle them on a few different repos.